atd-vzd/views/view_location_injry_count_cost_summary-schema.sql

```diff
@@ -24,25 +24,27 @@ SET row_security = off;
 --
 
 CREATE VIEW public.view_location_injry_count_cost_summary AS
-SELECT atcloc.location_id
-        , coalesce(ccs.total_crashes, 0) AS total_crashes
-        , coalesce(ccs.total_deaths, 0) AS total_deaths
-        , coalesce(ccs.total_serious_injuries, 0) AS total_serious_injuries
-        , coalesce(ccs.est_comp_cost, 0) AS total_deaths
-
-        FROM
-             atd_txdot_locations AS atcloc
-        LEFT JOIN (
-                SELECT atcloc.location_id,
-                         count(1)                           AS total_crashes,
-                         sum(atc.apd_confirmed_death_count) AS total_deaths,
-                         sum(atc.sus_serious_injry_cnt)     AS total_serious_injuries,
-                         sum(atc.est_comp_cost)             AS est_comp_cost
-                FROM (atd_txdot_crashes atc
-                           LEFT JOIN atd_txdot_crash_locations atcloc ON ((atcloc.crash_id = atc.crash_id)))
-                WHERE ((1 = 1) AND (atcloc.location_id IS NOT NULL) AND ((atcloc.location_id)::text <> 'None'::text))
-                GROUP BY atcloc.location_id
-        ) AS ccs ON (ccs.location_id = atcloc.location_id);
+
```

Comment:
It probably came from TablePlus, Postgres never changed the formatting before until now.

Reply:
I'm not sure tbh. TablePlus would make sense bc I really like clicking the "Beautify" button. But it could also be the syntax linter plugin for SQL in VSCode.